tests/xgboost/test_xgboost_autolog.py

```diff
@@ -730,12 +708,11 @@ def test_xgb_api_autolog_registering_model(bst_params, dtrain):
 @pytest.mark.parametrize("model_format", ["xgb", "json", "ubj"])
 def test_xgb_autolog_with_model_format(bst_params, dtrain, model_format):
     mlflow.xgboost.autolog(log_models=True, model_format=model_format)
-    with mlflow.start_run() as run:
+    with mlflow.start_run():
         xgb.train(bst_params, dtrain)
-    run_id = run.info.run_id
-    client = MlflowClient()
-    artifacts = [f.path for f in client.list_artifacts(run_id, "model")]
-    assert f"model/model.{model_format}" in artifacts
+    model = mlflow.last_logged_model()
+    artifacts = [f.path for f in mlflow.artifacts.list_artifacts(model.model_uri)]
```

Comment:
We can also use `list_logged_model_artifacts.`

Reply:
Do we have any preference on this?

Reply:
Got it. Btw, the API doc for `mlflow.artifacts.list_artifacts` implies it can take `model:/` uris.
https://mlflow.org/docs/latest/python_api/mlflow.artifacts.html?highlight=list_artifacts#mlflow.artifacts.list_artifacts